src/pages/whats-new/index.mdx

```diff
@@ -1,62 +1,94 @@
+We have a new library with over 400 pictograms available to use and even more to come in the next release. Along with the library comes an [.ai master file](https://github.com/carbon-design-system/carbon/raw/master/packages/pictograms/master/pictogram-master.ai) with all the pictograms organized by category. This library and master file will serve as the single source of truth representing the latest and approved pictograms. If you need pictograms or want to contribute, please follow the guidance outlines [here](https://www.ibm.com/design/language/iconography/pictograms/contribute/) and check out the [library](https://www.ibm.com/design/language/iconography/pictograms/library/).<br/>— <br/>9 October, 2019
 
 ## Search and filtering
 
 <Row>
 <Column colMd={6} colLg={8}>
 
-![Exporting stroke app icons: step 3](/images/updates_search_filter.png)
+[![link to UI icon library](./images/updates_search_filter.png)](https://www.ibm.com/design/language/iconography/ui-icons/library)
 
 </Column>
 </Row>
 
-This cycle, we released a major enhancement to our UI Icon and Pictogram libraries by adding search and category filtering, making it much easier to find what you’re looking for! We plan to make this component sticky in Q4 2019.<br/> — <br/>9 October, 2019
+This cycle, we released a major enhancement to our [UI Icon](https://www.ibm.com/design/language/iconography/ui-icons/library) and [Pictogram](https://www.ibm.com/design/language/iconography/ui-icons/library) libraries by adding search and category filtering, making it much easier to find what you’re looking for!<br/> — <br/>9 October, 2019
 
 
 ## Color palette V2
 
 <Row>
 <Column colMd={6} colLg={8}>
 
-![Exporting stroke app icons: step 3](/images/color.svg)
+[![link to color overview](./images/color.svg)](https://www.ibm.com/design/language/color/)
 
 </Column>
 </Row>
 
-We’re pleased to announce the new IBM Design Language V2 color palette. The updates from V1 are quite subtle at first glance but incorporate significant refinements when considering contrast ratios and accessibility for digital screens and interfaces. For an in-depth article outlining the methodology and process please visit [Because, colors are beautiful](https://medium.com/carbondesign/because-colors-are-beautiful-52dd4cc39f09), and find the details on our [color page in IDL](https://www.ibm.com/design/language/elements/color).
+We’re pleased to announce the new IBM Design Language [V2 color palette](https://www.ibm.com/design/language/color/). The updates from V1 are quite subtle at first glance but incorporate significant refinements when considering contrast ratios and accessibility for digital screens and interfaces. For an in-depth article outlining the methodology and process please visit [Because, colors are beautiful](.https://medium.com/carbondesign/because-colors-are-beautiful-52dd4cc39f09).
```

Comment:
```suggestion
We’re pleased to announce the new IBM Design Language [V2 color palette](https://www.ibm.com/design/language/color/). The updates from V1 are quite subtle at first glance but incorporate significant refinements when considering contrast ratios and accessibility for digital screens and interfaces. For an in-depth article outlining the methodology and process please visit [Because, colors are beautiful](https://medium.com/carbondesign/because-colors-are-beautiful-52dd4cc39f09).
```